layers/+tools/lsp/README.org

```diff
@@ -55,21 +59,99 @@ A number of configuration variables have been exposed via the LSP layer =config.
-| =lsp-ui-doc-enable=             | t       | When non-nil, the documentation overlay is displayed                                      |
-| =lsp-ui-doc-include-signature=  | nil     | When nil, signature omitted from lsp-ui-doc overlay (this is usually redundant)           |
-| =lsp-ui-sideline-enable=        | t       | When non-nil, the symbol information overlay is displayed                                 |
-| =lsp-ui-sideline-show-symbol=   | nil     | When non-nil, the symbol information overlay includes symbol name (redundant for c-modes) |
+| Variable name                      | Default | Description                                                                               |
+|------------------------------------+---------+-------------------------------------------------------------------------------------------|
+| =lsp-headerline-breadcrumb-enable= | nil     | When non-nil, shows breadcrumb on headerline.                                             |
+| =lsp-lens-enable=                  | nil     | When non-nil, shows code lens when it's supported.                                        |
+| =lsp-modeline-diagnostics-enable=  | t       | When non-nil, shows error diagnostics in modeline.                                        |
+| =lsp-modeline-code-actions-enable= | t       | When non-nil, shows available code actions in modeline.                                   |
+| =lsp-navigation=                   | `both'  | `simple' or `peek' to bind only xref OR lsp-ui-peek navigation functions                  |
+| =lsp-ui-remap-xref-keybindings=    | nil     | When non-nil, xref key bindings remapped to lsp-ui-peek-find-{definition,references}      |
+| =lsp-ui-doc-enable=                | t       | When non-nil, the documentation overlay is displayed                                      |
+| =lsp-ui-doc-include-signature=     | nil     | When nil, signature omitted from lsp-ui-doc overlay (this is usually redundant)           |
+| =lsp-ui-sideline-enable=           | t       | When non-nil, the symbol information overlay is displayed                                 |
+| =lsp-ui-sideline-show-symbol=      | nil     | When non-nil, the symbol information overlay includes symbol name (redundant for c-modes) |
+
+** Code Lens
+To always display code lens,
+#+BEGIN_SRC emacs-lisp
+(setq-default dotspacemacs-configuration-layers
+              '((lsp :variables lsp-lens-enable nil)))
+#+END_SRC
+
+
+
+
+This doesn't have any effect when code lens is not supported by current language server.
+
+** Error statistics on modeline
+By default, all error statistics of a project is displayed in the modeline.
+To disable this feature, set =lsp-modeline-diagnostics-enable= to =nil=.
+
+#+BEGIN_SRC emacs-lisp
+(setq-default dotspacemacs-configuration-layers
+              '((lsp :variables lsp-modeline-diagnostics-enable nil)))
+#+END_SRC
+
+
+To only display errors for the current file, you can set =lsp-modeline-diagnostics-scope= to =:file=.
```

Comment:
Mentioning the default for this var would be great, either mention it here, or have it in the configuration table. Either way users should see that the default is project wide.